src/dispatch/feedback/views.py

```diff
@@ -0,0 +1,97 @@
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+from sqlalchemy.orm import Session
+
+from dispatch.database import get_db, search_filter_sort_paginate
+
+from .models import (
+    FeedbackCreate,
+    FeedbackPagination,
+    FeedbackRead,
+    FeedbackUpdate,
+)
+from .service import create, delete, get, update
+
+
+router = APIRouter()
+
+
+@router.get("/", response_model=FeedbackPagination)
+def get_feedbacks(
```

Comment:
I see, you're using the method name to remind you that you're getting multiple feedback objects, not a single piece of feedback. Carry on then.